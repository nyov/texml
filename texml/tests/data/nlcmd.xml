<TeXML>
	Use <cmd name="it"/> italic
	or <cmd name="bf" nl="1"/> bold
	or <cmd name="tt" nl="0"/> tt.
</TeXML>
