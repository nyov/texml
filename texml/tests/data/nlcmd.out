Use \it  italic or \bf
bold or \tt  tt.
